test/groovy/ToolValidateTest.groovy

```diff
@@ -28,62 +28,40 @@ class ToolValidateTest extends BasePipelineTest {
                                       .around(thrown)
                                       .around(jlr)
 
-
     private static home
+    private static mtaJar
+    private static neoExecutable
+    private static cmCliExecutable
+
 
     private toolValidateScript
+    private commonPipelineEnvironment
 
 
     @BeforeClass
     static void createTestFiles() {
 
         home = "${tmp.getRoot()}"
         tmp.newFile('mta.jar')
+        tmp.newFolder('tools')
+        tmp.newFolder('bin')
+        tmp.newFile('tools/neo.sh')
+        tmp.newFile('bin/cmclient')
+
+        mtaJar = "$home/mta.jar"
+        neoExecutable = "$home/tools/neo.sh"
+        cmCliExecutable = "$home/bin/cmclient"
     }
 
     @Before
     void init() {
 
+        helper.registerAllowedMethod('fileExists', [String], { s -> return true })
+
         binding.setVariable('JAVA_HOME', home)
 
         toolValidateScript =  loadScript('toolValidate.groovy').toolValidate
-    }
-
-
-    @Test
-    void nullHomeTest() {
```

Comment:
Tests here removed since the prerequisite checks in the examinee has been removed. What was in principle wrong the the prerquisite checks?

Reply:
That now, home can be null, because it is possible to reuse the general configuration
e.g.
general:
mtaJarLocation: '/etc/mta'
neoHome: '/etc/neo-java-web-sdk'
cmCliHome: '/etc/cm-cli'

with the following command:

toolValidate tool:'mta'

Reply:
What is the reason for having this in the general section? Isn't this step specific. Is there a re-use of the properties (e.g. ```neoHome```) by other steps. Why should other steps be interested in ```neoHome```. Wouldn't this be a design smell?

Reply:
neoHome can be used by toolValidate and neoDeploy.